### DISCUSSION TITLE
Can we have a flux image to image workflow

### DISCUSSION THREAD

**Discussion by panorios** [Acly/krita-ai-diffusion#1021] [Ideas]
Or a basic flux generation based on the schnell model?
Thank you.

**Comment by arnonmoscona** [comment]
I would add that even instructions on how to do this manually would be great

**Comment by logikstate** [comment]
flux looks like its the next big thing after stable diffusion... so two thumbs up from me :)

**Comment by Acly** [comment]
There's an initial implementation, see https://github.com/Acly/krita-ai-diffusion/issues/1023#issuecomment-2269097778

**Comment by nesmonmehdydeal** [comment]
Hello, has FLux been added to Krita?

**Reply by nesmonmehdydeal** [reply]
Thanks, I saw it. [https://github.com/Acly/krita-ai-diffusion/releases/download/v0.1.0/krita_ai_diffusion-1.22.0-pre1.zip](https://github.com/Acly/krita-ai-diffusion/releases/download/v0.1.0/krita_ai_diffusion-1.22.0-pre1.zip)

**Reply by logikstate** [reply]
how did you find this? when I go to releases it only shows the main releases with the latest being from two weeks ago?

**Reply by nesmonmehdydeal** [reply]
https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.22.0

**Comment by Drakosha405** [comment]
https://github.com/XLabs-AI/x-flux

[flux-controlnet-canny](https://huggingface.co/XLabs-AI/flux-controlnet-canny)
[flux-RealismLora](https://huggingface.co/XLabs-AI/flux-RealismLora)

Near Updates
We are working on releasing new ControlNet weight models for Flux: OpenPose, Depth and more!
Stay tuned with [XLabs AI](https://github.com/XLabs-AI) to see IP-Adapters for Flux.

**Reply by dkqjhoifh** [reply]
I like it!!!!!

**Comment by nesmonmehdydeal** [comment]
Hi,

I would to know how to active:
"
Reference
Style
Composition
Face
Scribble
Line Art
Soft Edge
Canny Edge
Depth
Normal
Pose
Segment
Unblur
Stencil
Hands
"
with flux.

Also can you provide an option to use NF4 & GGUF on it, It is smaller for the same results. 

THANKS !

Please.
